packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -145,30 +163,50 @@ describeNoCompat("GC tombstone tests", (getTestObjectProvider) => {
             };
         };
 
+        let opCount = 0;
         const sendOpToUpdateSummaryTimestampToNow = async (container: IContainer) => {
+            const defaultDataObject = await requestFluidObject<ITestDataObject>(container, "default");
```

Comment:
It's hard to tell what is going on here. Can you please add some explanations here? Is this just a refactor?

Reply:
This is still just sending an op, the only advantage is that it's guaranteed to generate an op as the value for the field is always different "op" + some increment